### DISCUSSION TITLE
teslausb on raspi 2 w with 1 TB ssd

### DISCUSSION THREAD

**Discussion by shlomisaporta** [marcone/teslausb#807] [Q&A]
Hi , what is the recommended storage allocation for tesalcam folder in such a case?
 i have 50G music. should i give it all to the teslacam folder or should i split some of it to the raspi? which will give me better storage
thank you!!

**Comment by marcone** [comment]
The Raspberry Pi Zero 2 W can only use sd card for storage, not ssd.
I don't understand what you mean by "split some of it to the raspi".

**Comment by shlomisaporta** [comment]
Sorry , i was not clearHow should i split the sd card memory ?Lets say TeslaCam will be 500G - will there be raspberry use the other 500 or so ?Is there a use for that or should i put it all on TeslaCam?Sorry and thank youShlomi‫ב-10 בינו׳ 2024, בשעה 0:22, ‏‏marcone ***@***.***> כתב/ה:‬﻿
The Raspberry Pi Zero 2 W can only use sd card for storage, not ssd.
I don't understand what you mean by "split some of it to the raspi".

**Comment by marcone** [comment]
Since you have such a large sd card, I would set both CAM_SIZE and MUSIC_SIZE to 64GB.
Note that it's going to take a long time to copy that much music onto the device.

**Comment by shlomisaporta** [comment]
‫thank you very much, i will!What will be the use of the other part of the memory which i don't allocate ?‬﻿
Since you have such a large sd card, I would set both CAM_SIZE and MUSIC_SIZE to 64GB.
Note that it's going to take a long time to copy that much music onto the device.

**Reply by marcone** [reply]
The remaining space will be used by TeslaUSB to extend the car's 1 hour revolving window. A 1 TB card should have enough space to store several days worth of continuous recordings.
